python/semantic_kernel/connectors/memory/azure_cognitive_search/azure_cognitive_search_memory_store.py

```diff
@@ -83,6 +84,7 @@ async def create_collection(
         self,
         collection_name: str,
         vector_config: Optional[HnswVectorSearchAlgorithmConfiguration] = None,
+        search_resource_encryption_key: SearchResourceEncryptionKey = None,
```

Comment:
Tiny comment: I'd love to see the Optional type hint here for type checking, to keep it consistent with the optional type hint for the parameter before it.

Reply:
hHello @moonbox3  I have added the optional type hint !